### DISCUSSION TITLE
Regarding 2023 GCE application result

### DISCUSSION THREAD

**Discussion by levxn** [community/community#50217] [GitHub Education] [answered]
I submitted my application for the Campus Expert program 2 weeks back. And as per the result announcement, it was earlier said that it will be announced on the 15th of March 2023. I did not receive the status of my application yet. I submitted the application with an email id which was not set as the primary mail id for my GitHub account at the beginning. And now I have made that mail id as my primary id. So just in case if my application was rejected considering this fact, I kindly request the team to revalidate my application again for another time considering that I have set the primary mail ID for my GitHub correctly this time.

**Comment by levxn** [comment]
And as in general doubt, is it compulsory that the applicant must have that mail id through which he/she has submitted the application to be the primary id of his/her GitHub account?

**Comment by lasr21** [comment, marked as answer]
Hi there! We wanted to let you know that an email has been sent out to everyone. We also wanted to remind you that you can check the status of your application on https://education.github.com/campus_experts/status. Wishing you all the best!
